.github/workflows/_build_test.yml

```diff
@@ -89,8 +83,6 @@ jobs:
           path: test/suites/**/results
 
   test-success:
-    # Post-test CI step - to check all tests succeeded
-    # Github Branch Protection rules require this to pass to allow merging
```

Comment:
I think this comment still makes sense. Otherwise the "true" step might be confusing.